test/integration/image-component/default/pages/on-loading-complete.js

```diff
@@ -32,6 +35,22 @@ const Page = () => (
         objectFit="contain"
       />
     </div>
+    <hr />
+    <ImageWithMessage
+      id="5"
+      src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAAFCAYAAACAcVaiAAAAE0lEQVR42mNk+P+/ngEKGMngAADDdwx3Uz/3AAAAAABJRU5ErkJggg=="
+      layout="fixed"
+      width={64}
```

Comment:
Should we test onLoadComplete is called here as well?

Reply:
The component `ImageWithMessage` is using `onLoadingComplete` and it will print `loaded img6 with dimensions 3x`